saidump/saidump.cpp

```diff
@@ -10,18 +13,25 @@ extern "C" {
 #include "swss/table.h"
 #include "meta/sai_serialize.h"
 #include "sairediscommon.h"
+#include "swss/json.h"
+#include "swss/json.hpp"
 
 #include <getopt.h>
 
 // TODO split to multiple cpp
 
 using namespace swss;
+using json = nlohmann::json;
+
+// 100 MB
+constexpr int64_t RDB_FILE_MAX_SIZE = 1024 * 1024 * 100;
 
 struct CmdOptions
 {
     bool skipAttributes;
     bool dumpTempView;
     bool dumpGraph;
+    std::string rdbFile;
 };
 
 CmdOptions g_cmdOptions;
```

Comment:
please makr also this and below map as static, since you are already touching this file

Reply:
Sorry, I don't know what you're meaning.

Reply:
Do you want below:
```
static CmdOptions g_cmdOptions;
static std::map<sai_object_id_t, const TableMap*> g_oid_map;
```